### PR TITLE
[ADD] Live chat: participate

### PR DESCRIPTION
**Pull request by jero-odoo**
This PR is to separate the "Participate in Live Chat" section of the Live Chat basics doc into its own document.​

Doc tasks: ﻿[﻿https://www.odoo.com/mail/view?model=project.task&res_id=4014404&access_token=37c03da8-2edb-469c-a9cf-946e41a95932](https://www.odoo.com/mail/view?model=project.task&res_id=4014404&access_token=37c03da8-2edb-469c-a9cf-946e41a95932)

### CODE REVIEW

**Comment by jero-odoo** [comment]
@hojo-odoo  This is ready for review. Thanks!

File: content/applications/websites/livechat/participate.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+*Operators* are the users who respond to live chat requests from customers and website visitors. The
+information below outlines the necessary steps for operators participating in live chat
+conversations on an Odoo database.
+
+Set an online chat name
+-----------------------
+
+Before participating in a live chat, operators may choose to update their *Online Chat Name*. This
+is the name that is displayed to site visitors in the live chat conversation.
```
Comment (hojo-odoo) [inline content/applications/websites/livechat/participate.rst:13]:
```suggestion
is the name displayed to site visitors in the live chat conversation.
```

File: content/applications/websites/livechat/participate.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+   .. image:: participate/online-chat-name.png
+      :align: center
+      :alt: View of user profile in Odoo, emphasizing the Online Chat name field.
+
+Set online chat languages
+-------------------------
+
+If an operator speaks multiple languages, they can add this information to their user profile. This
+information can then be used to determine what conversations the operator is assigned. A visitor's
```
Comment (hojo-odoo) [inline content/applications/websites/livechat/participate.rst:36]:
```suggestion
information can then be used to determine which conversations the operator is assigned. A visitor's
```

File: content/applications/websites/livechat/participate.rst [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+language is determined via their browser's language settings.
+
+.. note::
+   Conversations are assigned to operators based on a number of criteria, including availability and
+   the number of ongoing conversations. While the operator's main language and additional languages
+   are taken into consideration, they do **not** supersede all other criteria.
```
Comment (hojo-odoo) [inline content/applications/websites/livechat/participate.rst:42]: Are there any criteria that they do supersede?

File: content/applications/websites/livechat/participate.rst [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+.. important::
+   Operators that do not show any activity in Odoo for more than thirty minutes are considered
+   disconnected, and subsequently removed from the channel.
+
+Manage live chat requests
+-------------------------
+
+When an operator is active in a channel, chat windows open in the bottom-right corner of the
+screen, no matter where they are in the database. They can participate in conversations without
+leaving their current location.
```
Comment (hojo-odoo) [inline content/applications/websites/livechat/participate.rst:75]:
```suggestion
screen, no matter where they are in the database. This allows them to access other pages and apps, while still participating in the conversation.
```

File: content/applications/websites/livechat/participate.rst [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+.. tip::
+   Conversations can also be accessed by clicking the :icon:`fa-comments` :guilabel:`(messages)`
+   icon in the menu bar.
+
+   .. image:: participate/menu-bar.png
+      :align: center
+      :alt: View of the menu bar in Odoo emphasizing the comments icon.
+
+Live chat conversations can also be viewed by navigating to :menuselection:`Discuss app`. New
```
Comment (hojo-odoo) [inline content/applications/websites/livechat/participate.rst:85]:
```suggestion
Live chat conversations can also be viewed by navigating to the :menuselection:`Discuss app`. New
```

**Review by hojo-odoo — APPROVED** [review]
Hey @jero-odoo,

Great job on this doc. Feel free to tag it for final review after you take a look at my suggestions.

Thanks,
John

**Comment by jero-odoo** [comment]
Thanks hojo!
@ksc-odoo this is ready for final review. Thanks!

File: content/applications/websites/livechat/participate.rst [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+information below outlines the necessary steps for operators participating in live chat
+conversations on an Odoo database.
+
+Set an online chat name
+-----------------------
+
+Before participating in a live chat, operators may choose to update their *Online Chat Name*. This
+is the name displayed to site visitors in the live chat conversation.
+
+To update the :guilabel:`Online Chat Name`, click on the user avatar in the top-right corner of any
```
Comment (ksc-odoo) [inline content/applications/websites/livechat/participate.rst:15]:
```suggestion
To update the *Online Chat Name*, click on the user avatar in the top-right corner of any
```

File: content/applications/websites/livechat/participate.rst [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+   .. image:: participate/online-chat-name.png
+      :align: center
+      :alt: View of user profile in Odoo, emphasizing the Online Chat name field.
+
+Set online chat languages
+-------------------------
+
+If an operator speaks multiple languages, they can add this information to their user profile. This
+information can then be used to determine which conversations the operator is assigned. A visitor's
+language is determined via their browser's language settings.
```
Comment (ksc-odoo) [inline content/applications/websites/livechat/participate.rst:37]:
```suggestion
language is determined, via their browser's language settings.
```

File: content/applications/websites/livechat/participate.rst [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+.. note::
+   Conversations are assigned to operators based on a number of criteria, including availability and
```
Comment (ksc-odoo) [inline content/applications/websites/livechat/participate.rst:40]:
```suggestion
   Conversations are assigned to operators, based on a number of criteria, including availability and
```

**Review by ksc-odoo — APPROVED** [review]
Hey @jero-odoo -- just finished my Final Review. Great work on this! Only a few comments that require your attention. once you've implemented the necessary modifications, feel free to tag this for Tech Review! 👍 Thanks

**Comment by jero-odoo** [comment]
thanks @ksc-odoo !
@samueljlieber this is ready for tech review!

**Review by samueljlieber — APPROVED** [review]
Looks good to me @jero-odoo! Nice job, I agree with you that moving this content to its own doc on participating in live chat makes sense. Thank you for work!
.....
@robodoo r+

**Pull request closed without merging**